README.md

```diff
@@ -1,23 +1,27 @@
 # analysis-workflows
 
-CWL workflow defintions for executing MGI analysis pipelines.
+This repo contains CWL workflow defintions for executing MGI analysis pipelines. 
 
-## Docker Images
+The structure of the repo is as follows:
+
+| Path | Description |
+| --- | --- |
+| definitions | parent directory containing all CWL tool and workflow definitions |
+| definitions/pipelines | all workflows which rely on subworkflows and tools to produce final outputs |
+| definitions/subworkflows | workflows that combine multiple tools to produce intermediate (used as inputs to other subworkflows) pipeline outputs |
+| definitions/tools | CWL that wrap command line interfaces or scripts connecting multiple tools |
+| example_data | example input data, input YAML files, and expected output files for testing |
 
-The master branch specifies `DockerRequirement` for each tool in the workflow.
+## CWL Documentation
 
-By contrast, the toil_compatibility branch relies on large, a/k/a fat, Docker images. Below is a mapping of workflow to Docker image name, all available on [DockerHub](https://hub.docker.com/u/mgibio/):
+For all documentation of pipelines, subworkflows and tools as well as additional information regarding test data, continous integration and configuration, please see the GitHub wiki:
+https://github.com/genome/analysis-workflows/wiki
 
-| Workflow | Docker Image | Main CWL |
-| --- | --- | --- |
-| unaligned_bam_to_bqsr | mgibio/dna-alignment | unaligned_bam_to_bqsr/workflow.cwl |
-| detect_variants | mgibio/cle | detect_variants/detect_variants.cwl |
-| rnaseq | mgibio/rnaseq | rnaseq/workflow.cwl |
-| tumor_only_exome | mgibio/cle | exome_workflow.cwl |
-| umi_alignment | mgibio/dna-alignment | umi_alignment/duplex_workflow.cwl |
+## Docker Images
 
-If a tool or workflow is a subworkflow of a larger workflow, then the same Docker image is required.
+All supported Docker images are available on mgibio DockerHub: https://hub.docker.com/u/mgibio/
 
+Many tools utilize Docker images hosted publically on DockerHub as well as [BioContainers](https://biocontainers.pro).
```

Comment:
I might say "hosted publicly in other repositories, such as Dockerhub or Biocontainers"

Reply:
A possible alternate phrasing of the whole sentence:
```
Many tools rely on third-party Docker images publicly available from sources such as Docker Hub and BioContainers.
```